modules/data/document/contrib/workflow/document_workflow.module

```diff
@@ -13,6 +13,7 @@ function document_workflow_entity_base_field_info(EntityTypeInterface $entity_ty
   if ($entity_type->id() === 'document') {
     $fields['state'] = BaseFieldDefinition::create('state')
       ->setLabel(new TranslatableMarkup('Status'))
+      ->setSetting('workflow', 'generation')
```

Comment:
Because the generation workflow is provided by the document_generation module, this needs to go in an alter hook in the document_generation module.

Reply:
Done.